src/config/config.node.ts

```diff
@@ -15,6 +15,9 @@ const NodeConfig: QuaggaJSConfigObject = {
     },
     locate: true,
     numOfWorkers: 0,
```

Comment:
There shouldn't be any numOfWorkers parameters left anywhere, so remove this one